tests/nimdoc/trunnableexamples.nim

```diff
@@ -1,19 +1,31 @@
 discard """
-cmd: "nim doc --doccmd:-d:testFooExternal --hints:off $file"
+cmd: "nim doc --doccmd:--hints:off --hints:off $file"
 action: "compile"
+nimoutFull: true
```

Comment:
without nimoutFull, this test wouldn't be sufficient to check that the bugs were fixed; so because of https://github.com/nim-lang/Nim/issues/18077 i have to split the test in 2 files temporarily until https://github.com/nim-lang/Nim/issues/18077 gets fixed